### PR TITLE
consolidating routes

### PR DESCRIPTION
**Pull request by mmcquillan**

### CODE REVIEW

File: core/publishers.go [hunk 1]
```diff
@@ -12,18 +12,31 @@ func Publishers(publishMsgs <-chan models.Message, config *models.Config) {
...
-					if connector.Active {
-						if sendToConnector(connector.ID, route.Connectors) {
-							if connector.Debug {
-								log.Print("Broadcasting to " + connector.ID + " (type:" + connector.Type + ") for route " + route.Connectors)
-								log.Printf("Message: %+v", message)
-								log.Print("")
+		for _, route := range config.Routes {
+			for _, m := range route.Matches {
+				match := true
+				match, _ = parse.Match(m.Message, message.Out.Text+" "+message.Out.Detail)
```
Comment (kcwinner) [inline core/publishers.go:18]: Can we simplify this logic? Or is this for readability?

Comment (mmcquillan) [reply core/publishers.go:18]: This was for readability, but what do you have in mind?

Comment (kcwinner) [reply core/publishers.go:18]: That's fine then. Just felt like it was something that could be refactored/condensed :)

Comment (mmcquillan) [reply core/publishers.go:18]: I did fix it over the weekend to pass in the match by reference so as to read slightly better.

**Pull request merged**